packages/react-reconciler/src/ReactDebugFiberPerf.js

```diff
@@ -231,6 +232,25 @@ export function recordScheduleUpdate(): void {
   }
 }
 
+export function startRequestCallbackTimer(): void {
+  if (enableUserTimingAPI && !isWaitingForCallback) {
+    isWaitingForCallback = true;
+    beginMark('(Waiting for async callback...)');
+  }
+}
+
+export function stopRequestCallbackTimer(didExpire: boolean): void {
+  if (enableUserTimingAPI) {
+    isWaitingForCallback = false;
+    const warning = didExpire ? 'Async work expired' : null;
```

Comment:
Not very obvious to me what this means. Is there a way to phrase it in a more user-accessible way?

Reply:
Yeah, let's think of something better. I was trying to keep it concise. What it means is:

"An async update starved because the main thread was blocked"

Reply:
What if it just said "Warning: Main thread is blocked"?